internal/ingress/controller/template/template.go

```diff
@@ -404,6 +404,14 @@ func buildProxyPass(host string, b interface{}, loc interface{}, dynamicConfigur
 		}
 	}
 
+	if location.ProxyPass.Address != "" || location.ProxyPass.Port != "" {
```

Comment:
Would you ever use a port alone?

Reply:
I figured that the address could be omitted if `ProxyToLocalNode` was set along with the port.